File: 03.business-os/NameCardManager/android/app/src/main/java/com/lsam/NameCardManager/commonosconsumer/BusinessOSCommonOSConsumerRegistry.java
package com.lsam.NameCardManager.commonosconsumer;

public final class BusinessOSCommonOSConsumerRegistry {
    public BusinessOSCommonOSConsumerDescriptor createDefault() {
        return new BusinessOSCommonOSConsumerDescriptor(
                "NameCardManager",
                "shared_provider",
                "os_side_consumer",
                "~/03.civilization-development/03.business-os/_commonos",
                "CommonOS",
                "NameCardManager",
                "offline-first"
        );
    }
}

File: 03.business-os/NameCardManager/iphone/NameCardManager/BusinessOSCommonOSConsumer/BusinessOSCommonOSConsumerRegistry.swift
import Foundation

enum BusinessOSCommonOSConsumerRegistry {
    static func createDefault() -> BusinessOSCommonOSConsumerDescriptor {
        BusinessOSCommonOSConsumerDescriptor(
            appName: "NameCardManager",
            providerRole: "shared_provider",
            consumerRole: "os_side_consumer",
            consumerRoot: "~/03.civilization-development/03.business-os/_commonos",
            uiOwner: "CommonOS",
            businessOwner: "NameCardManager",
            syncMode: "offline-first"
        )
    }
}
